R/data-survey.R

```diff
@@ -269,3 +288,50 @@ calc_survey_hiv_indicators <- function(survey_meta,
 
   val
 }
+
+
+#' Find Calendar Quarter Midpoint of Two Dates
+#'
+#' @param start_date vector coercibel to Date
+#' @param end_date vector coercibel to Date
+#'
+#' @return A vector of calendar quarters
+#'
+#' @examples
+#' start <- c("2005-04-01", "2010-12-01", "2016-01-01")
+#' end <-c("2005-08-01", "2011-05-01", "2016-06-01")
+#'
+#' @export
+get_mid_calendar_quarter <- function(start_date, end_date) {
```

Comment:
This needs tests, being both easily testable and full of modulo arithmetic.

Reply:
done in 33e2657